README.md

```diff
@@ -57,10 +57,10 @@ The test suite may be run locally by way of some simple composer scripts:
 
 ### Releasing
 
-To release a new version of the Update Tool, run:
+To release a new version of the Update Tool, simply create a new tag at the appropriate version. This will trigger the tests to run again. Assuming the tests pass, that send a release dispatch that triggers another GitHub Action to publish the release and upload the `update-tool.phar` to the release.
 
-- `composer release`
+Rebuild [pantheon-systems/docker-updatinator](https://github.com/pantheon-systems/docker-updatinator) to deploy a new version of the tool to the automation processes.
```

Comment:
Rebuild where/how? Rebuild in github? Rebuild in Quay?

Reply:
I believe so. This was copy/pasta from the original notes.